i18n.json.default.json

```diff
@@ -21,6 +21,7 @@
     "auth-admin-created": "CouchDB admin created",
     "auth-change-password": "Your password has been updated.",
     "auth-admin-creation-failed-prefix": "Could not create admin.",
-    "auth-passwords-not-matching": "Passwords do not match."
+    "auth-passwords-not-matching": "Passwords do not match.",
+    "create-db-partitioned-help": "A partitioned database requires a partition key for every document, where the document _id format is '&lt;partition_key&gt;:&lt;doc_key&gt;'. A partition is a logical grouping of documents. Use of partition queries can significantly improve response times over global queries."
```

Comment:
`Use of partition queries can significantly improve response times over global queries.` is a bit confusing to me.
I think we need to reword that.